features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/SyslogReceiverNioThreadPoolImpl.java

```diff
@@ -72,6 +77,21 @@
      * calls on the syslog port.
      */
     public static final int SOCKET_RECEIVER_COUNT = Runtime.getRuntime().availableProcessors() * 2;
+    
```

Comment:
Please undo this block of changes. We don't need the EVENT_PARSER_THREADS and EVENT_SENDER_THREADS constants in this class.